concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+ID	NUMBER	SPANISH	GLOSS	SWADESH_ROWE	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Mountain-1978-267-1	1	lengua	tongue	1	1205	TONGUE
+Mountain-1978-267-2	2	boca	mouth	2	674	MOUTH
+Mountain-1978-267-3	3	labio	lip	3	478	LIP
+Mountain-1978-267-4	4	diente	tooth	4	1380	TOOTH
+Mountain-1978-267-5	5	nariz	nose	5	1221	NOSE
+Mountain-1978-267-6	6	ojo	eye	6	1248	EYE
+Mountain-1978-267-7	7	oreja	ear	7	1247	EAR
+Mountain-1978-267-8	8	cabeza	head	8	1256	HEAD
+Mountain-1978-267-9	9	frente	forehead	9	123	FOREHEAD
+Mountain-1978-267-10	10	cabello	hair	10	1040	HAIR
+Mountain-1978-267-11	11	mentón	chin	11	1510	CHIN
+Mountain-1978-267-12	12	barba	beard	12	678	BEARD
+Mountain-1978-267-13	13	cuello	neck	13	1333	NECK
+Mountain-1978-267-14	14	teta, seno, pecho	woman’s breast	15	1402	BREAST
+Mountain-1978-267-15	15	espalda	back	17	1291	BACK
+Mountain-1978-267-16	16	hombro	shoulder	18	1482	SHOULDER
+Mountain-1978-267-17	17	brazo	arm	19	1673	ARM
+Mountain-1978-267-18	18	brazo superior	upper arm	20	431	UPPER ARM
+Mountain-1978-267-19	19	codo	elbow	21	981	ELBOW
+Mountain-1978-267-20	20	mano	hand	22	1277	HAND
+Mountain-1978-267-21	21	dedo de la mano	finger	23	1303	FINGER
+Mountain-1978-267-22	22	uña	fingernail	24	1258	FINGERNAIL
+Mountain-1978-267-23	23	pierna	leg	25	1297	LEG
+Mountain-1978-267-24	24	muslo	thigh	26	800	THIGH
+Mountain-1978-267-25	25	rodilla	knee	27	1371	KNEE
+Mountain-1978-267-26	26	espinilla	shin	28	186	SHIN
+Mountain-1978-267-27	27	pie	foot	29	1301	FOOT
+Mountain-1978-267-28	28	dedo del pie	toe	30	1389	TOE
+Mountain-1978-267-29	29	piel	skin	31	763	SKIN
+Mountain-1978-267-30	30	hueso	bone	32	1394	BONE
+Mountain-1978-267-31	31	sangre	blood	33	946	BLOOD
+Mountain-1978-267-32	32	corazón	heart	34	1223	HEART
+Mountain-1978-267-33	33	pene	penis	36	1222	PENIS
+Mountain-1978-267-34	34	hombre	man	38	1554	MAN
+Mountain-1978-267-35	35	varón	man	39	2106	MALE PERSON
```

Comment:
I don't know. Except for Siona, the terms elicited for the languages in the Mountain list are identical for hombre and varón.

Reply:
I tend to say: varón is more frequently used to denote "male person", grammatically, but you would translate them to English as "male": So if one says about a baby "es un varón", this can be interpreted to mean "it is male", but in German you would say "es ist ein Junge", and in English "it's a boy", etc. I haven't encountered an adjective reading so far, so I would say MALE PERSON makes more sense.